src/main.rs

```diff
@@ -151,12 +168,16 @@ fn backend(backends: &[Box<dyn Backend>]) -> &dyn Backend {
     }
 }
 
-fn exec(backends: &[Box<dyn Backend>], opts: Exec) -> Result<()> {
-    let backend = backend(backends);
+#[inline]
+fn workldr(workldrs: &[Box<dyn Workldr>]) -> &dyn Workldr {
+    // NOTE: this is stupid, but we only have one workldr, so... ¯\_(ツ)_/¯
+    &*workldrs[0]
+}
 
-    let map = mmarinus::Kind::Private.load::<mmarinus::perms::Read, _>(&opts.code)?;
+fn exec(backend: &dyn Backend, shim: impl AsRef<[u8]>, exec: impl AsRef<[u8]>) -> Result<()> {
+    //let map = mmarinus::Kind::Private.load::<mmarinus::perms::Read, _>(&opts.code)?;
```

Comment:
does it need to stay as a comment here?

Reply:
Heh oh yeah - I actually had a fixup patch that dropped this line.. and then I decided to leave it in just to see if any reviewer(s) would notice. So yeah, I'll make sure to delete that, and I award you with a gold star for Excellence in Code Review: ⭐